compiler/circle-execution-plan/src/CircleExecutionPlan.cpp

```diff
@@ -42,6 +42,13 @@ int entry(int argc, char **argv)
     .required(false)
     .default_value(false)
     .help("Plan with or without dsp (now can be used only with cmsisnn)");
+  arser.add_argument("--generate_json")
```

Comment:
Maybe we can use `--save_allocations`/`--save_tracealloc` as an option name? It probably will reflect better what we trying to output here.

Reply:
Done